### DISCUSSION TITLE
Welcome to ArrowOS_gsi Discussions!

### DISCUSSION THREAD

**Discussion by naz664** [naz664/ArrowOS_gsi#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PowerPlop** [comment]
@naz664 are you still planning sync with td sources?

**Comment by naz664** [comment]
No more plans to update A13 for now.
If A14 arrive, will check.
